presto-testing/src/main/java/io/prestosql/testing/DistributedQueryRunner.java

```diff
@@ -554,18 +554,17 @@ public Builder setNodeCount(int nodeCount)
 
         public Builder setExtraProperties(Map<String, String> extraProperties)
         {
-            this.extraProperties = extraProperties;
+            this.extraProperties = new HashMap<>(extraProperties);
             return this;
         }
 
         /**
          * Sets extra properties being equal to a map containing given key and value.
```

Comment:
nitL May be `Add the given key and value to the extra properties` or remove the comment?

Reply:
Removed altogether.